Unix/samples/Providers/Indication/Alert/XYZ_DiskFault.c

```diff
@@ -33,7 +33,7 @@ MI_Uint32 MI_CALL TriggerIndication(
     CHECKR_POST_RETURN(context, r);
 
     /* set properties of indication */
-    r = XYZ_DiskFault_Set_detailmessage( &fault, MI_T("Disk fault messsage") );
+    r = XYZ_DiskFault_Set_detailmessage( &fault, MI_T("Disk fault  message") );
```

Comment:
I didn't mean to include this leading space (it's because of a change for `faul` elsewhere that I didn't properly handle).